finalProject/finalProject/settings.py

```diff
@@ -28,7 +28,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = os.getenv("DEBUG") == "True"
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ["*"]
```

Comment:
Zostawmy puste, każda powinna pamiętać o tych zmiennych środowiskowych i pliku .env
```suggestion
ALLOWED_HOSTS = []
```